src/core/client/stream/tabs/Comments/Comment/PermalinkButton/PermalinkButton.css

```diff
@@ -2,3 +2,7 @@
   width: 350px;
   max-width: 80%;
 }
+
+.icon {
+  text-decoration: none!important;
```

Comment:
Is the  `!important`(and the others) here really necessary? My quick tests seems to work also without it? Could be me though. Just checking ways not to use it because `!imporant` should only be a last resort.